inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/StreamTransformController.java

```diff
@@ -58,27 +59,28 @@ public Response<Integer> save(@Validated @RequestBody TransformRequest request)
     }
 
     @RequestMapping(value = "/list", method = RequestMethod.GET)
-    @ApiOperation(value = "Query stream transform list")
+    @ApiOperation(value = "Get stream transform list")
     public Response<List<TransformResponse>> list(@RequestParam("inlongGroupId") String groupId,
             @RequestParam("inlongStreamId") String streamId) {
         return Response.success(streamTransformService.listTransform(groupId, streamId));
     }
 
     @RequestMapping(value = "/update", method = RequestMethod.POST)
     @OperationLog(operation = OperationType.UPDATE)
-    @ApiOperation(value = "Modify stream source")
-    public Response<Boolean> update(@Validated @RequestBody TransformRequest request) {
-        return Response.success(
-                streamTransformService.update(request, LoginUserUtils.getLoginUserDetail().getUsername()));
+    @ApiOperation(value = "Update stream transform")
+    public Response<Boolean> update(@Validated(UpdateValidation.class) @RequestBody TransformRequest request) {
+        String operator = LoginUserUtils.getLoginUserDetail().getUsername();
+        return Response.success(streamTransformService.update(request, operator));
     }
 
     @RequestMapping(value = "/delete", method = RequestMethod.DELETE)
     @OperationLog(operation = OperationType.UPDATE)
-    @ApiOperation(value = "Modify stream source")
+    @ApiOperation(value = "Delete stream transform")
     public Response<Boolean> delete(@RequestParam("inlongGroupId") String groupId,
-            @RequestParam("inlongStreamId") String streamId, @RequestParam("transformName") String transformName) {
-        return Response.success(
-                streamTransformService.delete(groupId, streamId, transformName,
-                        LoginUserUtils.getLoginUserDetail().getUsername()));
+            @RequestParam("inlongStreamId") String streamId,
```

Comment:
Could add @NotBlank,the following validation only checks Null, '''' will bypass the validation. 
If you do this, remember to add @Validated to the class, otherwise the validation will not work

Reply:
Done.